src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/ModelBindingHelper.cs

```diff
@@ -849,10 +850,10 @@ public static T ConvertTo<T>(object value, CultureInfo culture)
         }
 
         /// <summary>
-        /// Converts the provided <paramref name="value"/> to a value of <see cref="Type"/> <param name="type"/>
+        /// Converts the provided <paramref name="value"/> to a value of <see cref="Type"/> <paramref name="type"/>
         /// using the <see cref="CultureInfo.InvariantCulture"/>.
         /// </summary>
-        /// <param name="value">The value to convert."/></param>
+        /// <param name="value">The value to convert.</param>
```

Comment:
Hope this was a recent typo. It's a bit of a concern the `xml-docs-test` didn't catch it otherwise. Glad the new compiler catches these issues.